app/controllers/member_authentications_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MemberAuthenticationsController < ApplicationController
+  before_action :redirect_signed_up_members
+
+  def new
+    @page = Page.find params[:page_id]
+    @title = I18n.t('member_registration.title')
+
+    ## FIXME seed and fetch from DB
+    #
+    view = File.read("#{Rails.root}/app/liquid/views/layouts/member-registration.liquid")
```

Comment:
yeah this should either go in the database or be a slim template. blocking a user request on reading from the file system also seems awful slow